containers.md

```diff
@@ -0,0 +1,96 @@
+Containers
+======================
+
+## Installing a container runtime
+
+> [!WARNING]  
+> Do not use Docker Desktop to install Docker on MacOS. [As of Aug 2021 we need a license to use Docker Desktop at the Guardian.](https://www.docker.com/blog/updating-product-subscriptions/)
+
+MacOS doesn't support Containers natively. This unfortunately means that at the moment we need to create a Linux VM in order to run any kind of Container on our devices.
+
+Fortunately, there exists a number of tools to help us setup a VM and use it seamlessly as if we were running containers on our host machine. Docker Desktop is one of such tools, but as previously mentioned we can't use that without a license.
```

Comment:
I think the recommendations repo should make a recommendation here! At the moment, is this listing options?

What are teams using already? I know we already have some Docker Desktop licences, should we double down on thay?